include/sound/sof/ext_manifest.h

```diff
@@ -61,8 +61,22 @@ enum sof_ext_man_elem_type {
 	SOF_EXT_MAN_ELEM_WINDOW			= SOF_IPC_EXT_WINDOW,
 	SOF_EXT_MAN_ELEM_CC_VERSION		= SOF_IPC_EXT_CC_INFO,
 	SOF_EXT_MAN_ELEM_DBG_ABI		= SOF_IPC_EXT_USER_ABI_INFO,
+	SOF_EXT_MAN_ELEM_CONFIG_DATA		= SOF_IPC_EXT_CONFIG_DATA,
+	SOF_EXT_MAN_ELEM_PLATFORM_CONFIG_DATA	= SOF_IPC_EXT_PLATFORM_CONFIG_DATA,
+
+};
+
+/* EXT_MAN_ELEM_PLATFORM_CONFIG_DATA elements identificators */
+enum sof_cavs_config_elem_type {
+	SOF_EXT_MAN_CAVS_CONFIG_CAVS_LPRO	= 1,
+	SOF_EXT_MAN_CAVS_CONFIG_LAST_ELEM,	/*< keep it at the end of enum list */
```

Comment:
@fredoh9 weshould move this to ext-manifest-intel.h

Reply:
without `SOF_EXT_MAN_CAVS_CONFIG_LAST_ELEM`, how linux driver may know total possible elements? I think we still need this. I will update this PR soon. We can discuss upon my latest change.